tools/generators/dependency-mismatch/index.ts

```diff
@@ -73,3 +84,11 @@ function getUpdatedDependencies(tree: Tree, dependencies: Record<string, string>
     return acc;
   }, dependencies);
 }
+
+function getProjectScope(project: ReturnType<typeof getProjectConfig>) {
```

Comment:
move this to utils?

Reply:
yeah I was hesitant about that :D , we can do that later if we find shared usage for this

Reply:
will do a follow - normalizing all tags and tweaking this to utils